calculator.js

```diff
@@ -0,0 +1,21 @@
+
+add = (a, b) => {
+    return a + b
+}
+
+substract = (a, b) => {
+    return a - b
+}
+
+multiply = (a, b) => {
+    return a * b
+}
+
+divide = (a, b) => {
+    return a / b
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/bakersen/code-quality-badges/pullRequest?prid=8111319)
```suggestion
  return a / b
```